### DISCUSSION TITLE
Translation & support

### DISCUSSION THREAD

**Discussion by npirtube** [lbbrhzn/ocpp#387] [General]
Hello,

I have made the translation of the en.json file but I don't know how I can share it with you.

On the other hand, I would like to offer you my help to understand the protocol and the functioning of OCPP, as I am involved in the management of EV chargers.

I would love to be part of this project and help as much as possible.

Best regards

**Comment by lbbrhzn** [comment]
Great to hear your are offering to help! 
If you would like to contribute something the best way is to fork the ocpp reposity, add your your new file and then create a pull request.
See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks for more info.

**Comment by npirtube** [comment]
I have made a request to add Spanish, I hope I got it right.

I will soon install a charger in my house to be able to test the functionality of the charger.

**Comment by lbbrhzn** [comment]
pull request #409 has been merged!
